packages/firestore/lib/index.d.ts

```diff
@@ -2105,6 +2106,16 @@ export namespace FirebaseFirestoreTypes {
      */
     Filter: typeof Filter;
 
+    /**
+     * Returns the `VectorValue` class.
+     */
+    VectorValue: typeof VectorValue;
+
+    /**
+     * Creates a new VectorValue from the provided numbers.
+     */
+    vector(values?: number[]): VectorValue;
```

Comment:
just noting all the places - I believe this should be `fromJSON` (or `Vector.fromJSON`, specifically)